src/Microsoft.AspNet.Mvc.Formatters.Json/Logging/JsonInputFormatterLoggerExtensions.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.AspNet.Mvc.Formatters.Json.Logging
+{
+    internal static class JsonInputFormatterLoggerExtensions
+    {
+        private static readonly Action<ILogger, string, Exception> _jsonInputFormatterCrashed;
+
+        static JsonInputFormatterLoggerExtensions()
+        {
+            _jsonInputFormatterCrashed = LoggerMessage.Define<string>(
+                LogLevel.Debug,
```

Comment:
Actually let's talk to @lodejard about the right level here. I'm not sure this topic has been figured out fully.